ironfish-cli/src/commands/wallet/burn.ts

```diff
@@ -67,12 +68,21 @@ export class Burn extends IronfishCommand {
       description:
         'The block sequence that the transaction can not be mined after. Set to 0 for no expiration.',
     }),
+    eligibility: Flags.boolean({
+      default: true,
+      allowNo: true,
+      description: 'check testnet eligibility',
+    }),
   }
 
   async start(): Promise<void> {
     const { flags } = await this.parse(Burn)
     const client = await this.sdk.connectRpc(false, true)
 
+    if (flags.eligibility) {
```

Comment:
I had only added it to the mint on the assumption that people would be generally running both commands and it would reduce API load.

Not strongly opinionated her just giving context

Reply:
oh yeah, that's a good reason. I think our API is performing OK now so it should be fine.